pkg/chartverifier/checks/checks.go

```diff
@@ -285,28 +292,34 @@ func ImagesAreCertified(opts *CheckOptions) (Result, error) {
 	return r, nil
 }
 
-func getImageParts(image string) ([]string, string, string) {
+func getImageParts(image string) ImageReference {
```

Comment:
how about to rename this function to `getImageReference` or `parseImageReference` to reflect better what it is done here?

Reply:
Done